packages/spor-bottom-drawer-react-native/src/SimpleDrawer.tsx

```diff
@@ -0,0 +1,44 @@
+import { Text } from "@vygruppen/spor-typography-react-native";
+import React from "react";
+import { BottomDrawer } from "./BottomDrawer";
+import { DrawerHeader } from "./DrawerHeader";
+
+type SimpleDrawerProps = {
+  children?: JSX.Element | JSX.Element[];
+  title?: string;
+  titleSize?: "sm" | "lg";
```

Comment:
Istedenfor å be folk spesifisere dette, beregn det basert på antall bokstaver f.eks. :) Det tror jeg du kan gjøre med `textAlign` også